src/components/NewsletterSignup/NewsletterSignup.stories.tsx

```diff
@@ -0,0 +1,216 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Box, VStack } from "@chakra-ui/react";
+import { withDesign } from "storybook-addon-designs";
+import NewsletterSignup from "./NewsletterSignup";
+import { sectionTypeArray } from "../../helpers/types";
+import Heading from "../Heading/Heading";
+
+const meta: Meta<typeof NewsletterSignup> = {
+  title: "Components/Form Elements/NewsletterSignup",
+  component: NewsletterSignup,
+  decorators: [withDesign],
+  parameters: {
+    jest: ["NewsletterSignup.test.tsx"],
+  },
+  argTypes: {
```

Comment:
Add `privacyPolicyLink` to the argTypes and make sure to show the default value.

Reply:
Added.

Reply:
Please also add the default value: https://www.nypl.org/help/about-nypl/legal-notices/privacy-policy

Reply:
It was a typo that it did not show up before. This is fixed now.